### PR TITLE
Create docker image for mpdev tool

### PR DESCRIPTION
**Pull request by gibbleyg**

### CODE REVIEW

File: cloudbuild.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+timeout: 1200s
+
+steps:
+- id: Docker push
```
Comment (huyhg) [inline cloudbuild.yaml:4]: s/push/build

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FROM gcr.io/cloud-builders/bazel
```
Comment (huyhg) [inline Dockerfile:1]: Let's give this stage a name instead of using its index on L13

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+COPY . .
+RUN bazel build --stamp --workspace_status_command="./scripts/workspace-status.sh" //mpdev:mpdev
+
+
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:slim
```
Comment (huyhg) [inline Dockerfile:6]: A quick comment on what we're bundling in this 2nd stage

File: mpdev/BUILD.bazel [hunk 4]
```diff
@@ -12,4 +12,4 @@ go_binary(
     name = "mpdev",
     embed = [":go_default_library"],
     visibility = ["//visibility:public"],
-)
```
Comment (huyhg) [inline mpdev/BUILD.bazel:15]: We should keep the newlines at end of file--here and others

**Review by huyhg — APPROVED** [review]

File: mpdev/BUILD.bazel (hunk 4 above)
Comment (gibbleyg) [reply mpdev/BUILD.bazel]: Oops done

**Pull request merged**